cloudevents/working-drafts/xml-format.md

```diff
@@ -85,11 +86,13 @@ Extension context attributes MUST be decorated with the appropriate CloudEvent t
 designators using an `xsi:type` XML attribute, this allows them to be exchanged
 without loss of type information.
 
-REQUIRED and OPTIONAL context attribute MAY be decorated with an`xsi:type`, if present this
-designator MUST match that of the type specified by the [CloudEvent context attributes][ce-attrs].
+An XML element representing a REQUIRED or OPTIONAL context attribute MAY be decorated with
+an `xsi:type` XML attribute. If present, this designator MUST match that of the type specified
+by the [CloudEvent context attributes][ce-attrs].
 
-No other XML element attributes are expected, if present they MUST be ignored during
-processing.
+An XML element representing a CloudEvent attribute MUST NOT contain any XML attributes
+other than `xsi:type` and attributes in the `xmlns` namespace. The XML element MUST NOT
```

Comment:
Note for discussion: what about the xml namespace, e.g. for `xml:space`?

Reply:
I'm not sure what the "`xmlns` namespace" is. Do you mean "the CloudEvents XML namespace" ?

Reply:
I mean as in it's okay to have:

```xml
<ce:xyz xsi:type="string" xmlns:xsi=xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" />
```

(Or even introduce the `ce` namespace alias in the same element.)

Reply:
True - and I would like to allow people to add extensions, even on the CE defined data. Just haven't found the right time to mention it....but I guess I just did :-)

Reply:
From w3.org:

> The prefix xmlns is used only to declare namespace bindings and is by definition bound to the namespace name http://www.w3.org/2000/xmlns/. It MUST NOT be declared . Other prefixes MUST NOT be bound to this namespace name, and it MUST NOT be declared as the default namespace. Element names MUST NOT have the prefix xmlns